1.	The time required for servicing transmissions is normally distributed with mean = 45 minutes and std = 8 minutes. The service manager plans to have work begin on the transmission of a customer’s car 10 minutes after the car is dropped off and the customer is told that the car will be ready within 1 hour from drop-off. What is the probability that the service manager cannot meet his commitment?



In [ ]:
#mean=45min
#std=8min
#buffer time=10 min
#commitment time=60 min
#we have to find p(x>(60-10))=1-p(x<(60-10))
from scipy import stats
prob=1-stats.norm.cdf(50,loc=45,scale=8)
prob

0.26598552904870054

2.	The current age (in years) of 400 clerical employees at an insurance claims processing center is normally distributed with mean mean = 38 and Standard deviation std =6. For each statement below, please specify True/False. If false, briefly explain why.  
*   More employees at the processing center are older than 44 than between 38
and 44.
*   A training program for employees under the age of 30 at the center would be expected to attract about 36 employees.



 **More employees at the processing center are older than 44 than between 38
and 44.**

In [ ]:
#n=400
#mean=38
#std=6
#we have to find p(x>44) and p(38<x<44) and compare the probability
from scipy import stats
prob_1=1-stats.norm.cdf(44,loc=38,scale=6)
prob_1

0.15865525393145707

In [ ]:
prob_2=stats.norm.cdf(44,loc=38,scale=6)-stats.norm.cdf(38,loc=38,scale=6)
prob_2

0.3413447460685429

prob_2 is greater than prob_1. Which implies that the statement is **FALSE**

**A training program for employees under the age of 30 at the center would be expected to attract about 36 employees.**

In [ ]:
#we have to find p(x<=36)
p=stats.norm.cdf(36,loc=38,scale=6)
p

0.36944134018176367

*here p=0.36944 which is less than 0.5. so it's unlikely that under the age of 30, the training programme would attract 36 employee.*

4)	Let X ~ N(100, 20^2). Find two values, a and b, symmetric about the mean, such that the probability of the random variable taking a value between them is 0.99.

1.   90.5, 105.9
2.   80.2, 119.8
3.   22, 78
4.   48.5, 151.5
5.   90.1, 109.9  



In [ ]:
#mean=100
#std=20
from scipy import stats
from scipy.stats import norm
stats.norm.cdf(105.9,loc=100,scale=20)-stats.norm.cdf(90.5,loc=100,scale=20)

0.2986095735052065

In [ ]:
stats.norm.cdf(119.8,loc=100,scale=20)-stats.norm.cdf(80.2,loc=100,scale=20)

0.6778258809783382

In [ ]:
stats.norm.cdf(78,loc=100,scale=20)-stats.norm.cdf(22,loc=100,scale=20)

0.13561796460236508

In [ ]:
stats.norm.cdf(151.5,loc=100,scale=20)-stats.norm.cdf(48.5,loc=100,scale=20)

0.9899759913364774

In [ ]:
stats.norm.cdf(109.9,loc=100,scale=20)-stats.norm.cdf(90.1,loc=100,scale=20)

0.37939987949102466

Answer: So, a=48.5 and b=151.5

5.)	Consider a company that has two different divisions. The annual profits from the two divisions are independent and have distributions Profit1 ~ N (5, 32) and Profit2 ~ N (7, 42) respectively. Both the profits are in Dollar Million. Answer the following questions about the total profit of the company in Rupees. Assume that $1 = Rs. 45

1.   Specify a Rupee range (centered on the mean) such that it contains 95% probability for the annual profit of the company.
2.   Specify the 5th percentile of profit (in Rupees) for the company
3.   Which of the two divisions has a larger probability of making a loss in a given year?


1)

In [ ]:
from scipy import stats
import numpy as np
profit_1_mean=5*45
profit_2_mean=7*45
profit_1_std=np.sqrt(32*45)
profit_2_std=np.sqrt(42*45)
total_profit_mean=profit_1_mean+profit_2_mean
total_profit_std=profit_1_std+profit_2_std

In [ ]:
confidance_interval=stats.norm.interval(0.95,loc=total_profit_mean,scale=total_profit_std)
confidance_interval

(380.416866596655, 699.5831334033451)

2)

In [ ]:
#fifth percentile
percentile_5=stats.norm.ppf(0.05,loc=total_profit_mean,scale=total_profit_std)
percentile_5

406.0736126534633

3) Here the mean of profit 1 and profit 2 are 5 and 7 in dollar million. And standard deviations are 5.66 and 6.48 approximately. So, it's clear that mean in larger for division 2 compared to division 1. Also, in division 1, standard deviation is larger than mean. But in division 2 , standard deviation is slightly smaller than mean. Therefore, Division 1 has a larger probability of making a loss in a given year because it has a lower mean profit and a higher standard deviation, indicating more variability and potential for negative outcomes.